### DISCUSSION TITLE
Editieren - Datumsformatierung in Eingabefeld

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#159] [Q&A]
Für's **Editieren** habe ich zwei Datumsfelder (Errichtungsdatum und Abbruchdatum) im CMS definiert. Wenn ich das Datum aus dem Datepicker auswähle, bekomme ich eine Fehlermeldung:

![datum_value_too_long](https://github.com/gis-eni/webgis-forum/assets/100088983/415c8880-8fd9-488f-81be-e8ab32206948)

In der Datenbank (PostGreS) sind die Felder als CHARACTER (10) definiert.
Damit das funktionieren kann, muss ich das Datum in die Form 'DD.MM.YYYY' bringen.
Allerdings ist mir nicht klar wo im CMS ich das in welcher Form definieren muss.

**Comment by gis-eni** [comment]
Ein Datum aus dem DatePicker ist immer länger, hier kommt auch noch die Uhrzeigt mit dazu. Datumsfelder müssen in der Regel auch in ein Datumsfeld geschrieben werden. Wenn du das als String verspeicherst, geht das nicht mit DatePicker!

Der Anwender muss das hier wohl so eingeben. Du kannst mit einer Reglar Expression validieren, ob die Eingabe dem Format entspricht.
